### ISSUE TITLE
Website at autumnai.com 

### ISSUE THREAD

**Issue by bitcointroy** [autumnai/autumnai.com#9]
I attempted to visit autumnai.com and instead saw some very spammy ads and redirects. I haven't looked into it thoroughly, but I thought you should be aware.

**Comment by hobofan** [comment]
@bitcointroy Thanks for the report! We were already alerted in another thread (https://github.com/rust-lang/rust-www/issues/976) and I've since tried to remove any clickable links to autumnai.com.

**Comment by simeg** [comment]
Because it leads to malware, consider making this repo private? Because it has the website URL as the name and it sounds like the website won't be up anytime soon.

**Comment by hobofan** [comment]
@simeg Good call. I renamed the repo, since I want it to be still available for people and we don't have private repos for the org.
